docs/publishing/sections/4_analytics_portfolio_site.md

```diff
@@ -9,3 +9,118 @@ For these situations, the [Jupyter Book-based](https://jupyterbook.org/en/stable
+            parts:
+            - caption: Introduction
+            - chapters:
+              - notebook: ./my-analyses/notebook_1.ipynb
+              - notebook: ./my-analyses/notebook_2.ipynb
+            ```
+## Building and Deploying your Report:
+### Build your Report:
+**Note:** The build command must be run from the root of the repo!
+1. Navigate back to the repo data-analyses and install the portfolio requirements with
+`pip install -r portfolio/requirements.txt`
+2. Then run `python portfolio/portfolio.py build my_report` to build your report
+    * The `my_report` will be replaced by the name of your `.yml` file in [data-analyses/portfolio/sites](https://github.com/cal-itp/data-analyses/tree/main/portfolio/sites).
+    * Your build will be located in: `data-analyses/portfolio/my_report/_build/html/index.html`
+4. Add the files using `git add` and commit your progress!
+
+
+### Deploy your Report:
+
+1. Make sure you are in the root of the data-analyses repo: `~/data-analyses`
+2. Run `python portfolio/portfolio.py build my_report --deploy`
+    * The `my_report` will be replaced by the name of your `.yml` file in [data-analyses/portfolio/sites](https://github.com/cal-itp/data-analyses/tree/main/portfolio/sites).
+3. Once this runs, you can check the preview link at the bottom of the output. It should look something like:
+    * `–no-deploy`: `file:///home/jovyan/data-analyses/portfolio/my-analysis/_build/html/index.html`
+    * `–deploy`: `Website Draft URL: https://my-analysis--cal-itp-data-analyses.netlify.app`
+4. Add the files using `git add` and commit!
+5. Your notebook should now be displayed in the [Cal-ITP Analytics Portfolio](https://analysis.calitp.org/)
+
+
+### Other Specifications:
+ * You also have the option to specify: run `python portfolio/portfolio.py build --help` to see the following options:
+     * `--deploy / --no-deploy`
+         * deploy this component to netlify.
+     * `--prepare-only / --no-prepare-only`
+         * Pass-through flag to papermill; if true, papermill will not actually execute cells.
+     * `--execute-papermill / --no-execute-papermill`
+         * If false, will skip calls to papermill
+     * `--no-stderr / --no-no-stderr`
+         * If true, will clear stderr stream for cell outputs
+     * `--continue-on-error / --no-continue-on-error`
+         * Default: no-continue-on-error
+
+
+## Adding to the Makefile:
+
+Another way to write to the Analytics Portfolio is to use the Makefile and run
+`make build_my_report -f Makefile` in data-analyses
+
+Example makefile in [`cal-tip/data-analyses`](https://github.com/cal-itp/data-analyses/blob/main/Makefile):
+
+```
+build_my_reports:
+    pip install -r portfolio/requirements.txt
+    git rm portfolio/my-analyses/ -rf
+    python portfolio/portfolio.py build my-analyses --deploy
+    git add portfolio/my-analyses/district_*/ portfolio/dla/*.yml portfolio/dla/*.md
+    git add portfolio/sites/my-analyses.yml
```

Comment:
switch out `my-analyses` for `my_report`, because it's done off of the name of yml file, which is `sites/my_report.yml`